resources/js/components/Room/MembersComponent.vue

```diff
@@ -329,21 +468,59 @@ export default {
 
       this.isLoadingAction = true;
 
-      Base.call('rooms/' + this.room.id + '/member/' + this.deleteMember.id, {
+      Base.call('rooms/' + this.room.id + '/member/' + this.removeMember.id, {
         method: 'delete'
       }).then(response => {
         // remove user entry from list
-        this.removeMemberItem(this.deleteMember);
+        this.removeMemberItems([this.removeMember.id]);
       }).catch((error) => {
         if (error.response.status === env.HTTP_GONE) {
-          this.removeMemberItem(this.deleteMember);
+          this.removeMemberItems([this.removeMember.id]);
         }
         Base.error(error, this.$root);
       }).finally(() => {
         this.$refs['remove-member-modal'].hide();
         this.isLoadingAction = false;
       });
     },
+    /**
+     * Remove multiple members
+     * @param bvModalEvt
+     */
+    confirmBulkRemoveMember: function (bvModalEvt) {
+      // prevent modal from closing
+      bvModalEvt.preventDefault();
+
+      this.isLoadingAction = true;
+
+      const toBeRemovedMembers = this.selectedMembers.map(user => user.id);
+
+      Base.call('rooms/' + this.room.id + '/member', {
+        method: 'delete',
+        data: { users: toBeRemovedMembers }
+      }).then(response => {
+        // remove user entry from list
+        this.removeMemberItems(toBeRemovedMembers);
+        this.$refs['bulk-remove-members-modal'].hide();
+      }).catch((error) => {
+        // removing failed
+        if (error.response) {
+          // failed due to form validation errors
+          if (error.response.status === env.HTTP_UNPROCESSABLE_ENTITY) {
+            this.errors = error.response.data.errors;
+            return;
+          }
+        }
+        if (error.response.status === env.HTTP_GONE) {
```

Comment:
This error code is never neved by the backend, this is only possible on a single removal